dotCMS/src/main/java/com/dotcms/security/apps/ParamDescriptor.java

```diff
@@ -3,13 +3,14 @@
 import com.dotmarketing.util.UtilMethods;
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.List;
 
 /**
  * This is an implementation of a Parameter
  * Class used to collect properties or secrets set from the front-end
  * This is mostly used to pass values from the front-end into the Resource.
  */
-public class ParamDescriptor extends AbstractProperty<String>{
+public class ParamDescriptor extends AbstractProperty<Object>{
```

Comment:
Codacy found an issue: [A class which only has private constructors should be final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5895122)